.python-version

```diff
@@ -0,0 +1 @@
+3.9
```

Comment:
`slim` isn't compatible with Python >= 3.10, so for the time being we need to pin the Python version